freetype/build.zig

```diff
@@ -67,11 +67,20 @@ pub fn build(b: *std.build.Builder) !void {
     main_tests.setBuildMode(mode);
     main_tests.setTarget(target);
     main_tests.addPackage(c_pkg);
+
+    // Remove once the stage2 compiler fixes pkg std not found
+    main_tests.addPackage(utils_pkg);
+
     main_tests.addPackage(pkg);
     link(b, main_tests, .{ .freetype = .{
         .ft_config_path = "./test/ft",
         .brotli = true,
     } });
+    var font_option = b.addOptions();
+
+    font_option.addOption([]const u8, "font", try std.fs.cwd().readFileAlloc(b.allocator, "upstream/assets/FiraSans-Regular.ttf", std.math.maxInt(u32)));
```

Comment:
You can just set ``exe.main_pkg_path`` which will solve the @embedFile issue

Reply:
So I add `main_tests.main_pkg_path = "upstream/assets/";` in build.zig and use `@embedFile("FiraSans-Regular.ttf")`?
It doesn't work so I don't think that's what you meant

Reply:
I meant setting ``main_pkg_path`` to thisDir() and then @embedFile("../upstream...")

Reply:
Ah yeah, that was it thanks